### PR TITLE
Fix cert and operator policy in compliance history

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Fix this https://redhat-internal.slack.com/archives/CVAB9H3DE/p1711973234038419

### CODE REVIEW

**Comment by mprahl** [comment]
@yiraeChristineKim I think you'll have the same issue I fixed in the Gatekeeper compliance history tests where we need to use policy to do the clean up since the tests apply to all managed clusters which is an unknown amount in a BVT run.

**Comment by mprahl** [comment]
> > @yiraeChristineKim I think you'll have the same issue I fixed in the Gatekeeper compliance history tests where we need to use policy to do the clean up since the tests apply to all managed clusters which is an unknown amount in a BVT run.
> 
> Ok... but how other tests are run?

I think most are scoped to just a self-managed hub (e.g. local-cluster) and optionally one or the managed cluster. The compliance history tests started using `verifyPolicyOnAllClusters` with placements that apply to all clusters so that our coverage of environments is better in BVT and/or QE runs of our tests.

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -690,21 +693,23 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 			Expect(err).ToNot(HaveOccurred())
 
 			// Delete subscription, catalog, etc at once
-			By("Deleting the namespace")
-			_, err = common.OcHub(
+			By("Deleting the namespace policy")
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:696]: I think the prune object behavior of the policy handles this now.

File: test/integration/compliance_history_test.go [hunk 2]
```diff
@@ -690,21 +693,23 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
 			)
 			Expect(err).ToNot(HaveOccurred())
 		})
 
 		By("Creating the namespace")
-		_, err := common.OcHub(
-			"create",
-			"ns",
-			opNs,
+		_, err := common.OcManaged(
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:708]: Did you mean to apply this to the hub since this is a policy?

Comment (yiraeChristineKim) [reply test/integration/compliance_history_test.go:708]: oh mistake Thanks

File: test/integration/compliance_history_test.go [hunk 3]
```diff
@@ -690,21 +693,23 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
```
Comment (mprahl) [inline test/integration/compliance_history_test.go:708]: @yiraeChristineKim this still needs to be updated to `OcHub`.

Comment (yiraeChristineKim) [reply test/integration/compliance_history_test.go:708]: why?!!

Comment (mprahl) [reply test/integration/compliance_history_test.go:708]: Why would you apply a policy on the managed cluster?

**Review by mprahl — APPROVED** [review]

**Pull request merged**